### PR TITLE
Reorganize Fastify table of contents [WIP]

### PR DESCRIPTION
**Pull request by Sambaboy**
changes...
re-org TOC with new chapter titles
add Terminology section with Fastify terms defined in Resources chapter

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: src/website/TOC [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// recommedations for table of contents; added new Overview paragraph; suggest you add a Terminology section defining things
```
Comment (jsumners) [inline src/website/TOC:1]: Is this an ignored comment? Isn't this document just Markdown? If so, I don't think it will be ignored.

**Review by jsumners — COMMENTED** [review]
Can you provide some motivation for this? How does it improve things? I don't see a comparison to the previous implementation.

File: src/website/TOC (hunk 1 above)
Comment (Sambaboy) [reply src/website/TOC:1]: sure, i'm a tech writer and i met Tech Director and he told me Fastify docs needed a fresh look.  Yeah, it's just Markdown b/c i'm a tech writer not a coder.  It's merely a launching pad to do a reorg.

Comment (jsumners) [reply src/website/TOC:1]: That's cool. If this is still a "work in progress" PR, just update the title to include "[WIP]".

Comment (Sambaboy) [reply src/website/TOC:1]: K, did that...tks

**Comment by Sambaboy** [comment]
My PR was by no means intended as a straight merge into the Fastify's Docs master repo.  It is merely guidelines to the docs' guru in rethinking how to organize.

**Comment by jsumners** [comment]
Feedback is of course welcome, but you'll find that it gets incorporated quicker (if at all) if you submit a PR that actually implements your ideas.

**Comment by Eomm** [comment]
Closing for https://github.com/fastify/website-next

**Pull request closed without merging**